liquid_benchmarks/benchmark_liquid.rb

```
@@ -0,0 +1,46 @@
+require 'benchmark/ips'
+require 'json'
+
+module Benchmark
+  module Liquid
+    def liquid(label=nil, time: 10, warmup: 5, &block)
+      unless block_given?
+        raise ArgumentError.new, "block should be passed"
```

I think it's okay to just let it blow up later if the block isn't given.